src/cli.rs

```diff
@@ -103,6 +103,9 @@ pub struct Opts {
     /// Which sudo command to use. Must accept at least two arguments: user name to execute commands as and the rest is the command to execute
     #[clap(long)]
     sudo: Option<String>,
+    /// Prompt for sudo password during activation.
+    #[clap(long)]
+    interactive_sudo: Option<bool>,
```

Comment:
May I ask you to also update https://github.com/serokell/deploy-rs/blob/master/interface.json to reflect the new option?

Reply:
Sure, pushing this now. I also noticed some other options may be missing from the schema, including `remoteBuild` and `sudo`.

Reply:
Indeed, good catch